doc/api/child_process.markdown

```diff
@@ -176,8 +176,8 @@ replace the existing process and uses a shell to execute the command.*
 
 ### child_process.execFile(file[, args][, options][, callback])
 
-* `file` {String} The filename of the program to run
-* `args` {Array} List of string arguments
+* `file` {String} An absolute path to an executable file.
```

Comment:
That's not true, don't know what gives @ryansobol that idea:

```
> child_process.execFile('date', function(_, stdout){console.log(stdout)});null;
null
> Thu 31 Dec 2015 16:43:59 PST
```

man pages:

> The execlp(), execvp(), and execvpe() functions duplicate the actions of the shell in searching for an executable file if the specified filename does not contain a slash (/) character. The file is sought in the colon-separated list of directory pathnames specified in the PATH environment variable.

see http://linux.die.net/man/3/execvp